webrtc.html

```diff
@@ -556,49 +585,51 @@
               </li>
 
               <li>
-                <p>Add <var>stream</var> to <var>connection</var>'s <a href=
-                "#remote-streams-set">remote streams set</a>.</p>
+                <p>Create a new <code>RTCRtpReceiver</code> object <var>receiver</var>
+                  for <var>track</track>, and add it
+                  to <var>connection</var>'s <a href=
+                "#receivers-set">set of receivers</a>.</p>
               </li>
 
               <li>
-                <p><a href="#fire-a-stream-event">Fire a stream event</a> named
-                <code title="event-MediaStream-addstream"><a href=
-                "#event-mediastream-addstream">addstream</a></code> with
-                <var>stream</var> at the <var title="">connection</var>
-                object.</p>
+                <p><a href="#fire-a-track-event">Fire an event</a> named
+                <code title="event-track"><a href=
+                "#event-track">track</a></code> with
+                <var>receiver</var>, <var>track</var>, and <var>streams</var>
+                at the <var title="">connection</var> object.</p>
               </li>
             </ol>
           </li>
         </ol>
 
         <p>When a user agent has negotiated media for a component that belongs
-        to a media stream that is already represented by an existing
-        <code><a>MediaStream</a></code> object, the user agent MUST associate
-        the component with that <code><a>MediaStream</a></code> object.</p>
+        to a media track that is already represented by an existing
+        <code><a>MediaStreamTrack</a></code> object, the user agent MUST associate
+        the component with that <code><a>MediaStreamTrack</a></code> object.</p>
```

Comment:
I think we can't just s/MediaStream/MediaStreamTrack/ here. From the discussion in #3, I think we decided to remove this section.